com.photopea.Photopea.metainfo.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>com.photopea.Photopea</id>
+    <launchable type="desktop-id">com.photopea.Photopea.desktop</launchable>
+    <name>Photopea</name>
+    <summary>Free alternative for Adobe Photoshop users</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LicenseRef-proprietary</project_license>
```

Comment:
technically this is the license of the app, bot the web app that runs in the browser.

Reply:
This should be reconsidered, see this discussion:https://github.com/flathub/com.github.vikdevelop.photopea_app/issues/2#issuecomment-1660975290